CONTRIBUTING.md

```diff
@@ -0,0 +1,99 @@
+# Kubernetes in Rust : The contributing guide.
+
+Thank you for your interest in this project. In order to contribute to this project, please get familiar with the contributing rules and the git workflow.
+
+## Git workflow :
+
+![Git workflow](https://i.imgur.com/WMNLhY8.jpg)
+
+This project is organized within 2 repositories :
+- [The main repository](https://github.com/dev-sys-do/kudo)
```

Comment:
You should call that one the upstream repo.

Reply:
Fixed in the latest commit, thanks !